web/src/components/filtersform.js

```diff
@@ -276,15 +280,38 @@ class FiltersForm extends React.Component {
       {params.get('approvals') && this.props.showChangeParams ? ', approvals ' + params.get('approvals') : ''}
       {params.get('exclude_approvals') && this.props.showChangeParams ? ', excluded approvals named ' + params.get('exclude_approvals') : ''}
       {params.get('state') && this.props.showChangeParams ? ', change state ' + params.get('state') : ''}
+      {params.get("project_definition") ? ", change project_definition " + params.get("project_definition") : ""}
       {'.'}
     </b>
     return resume
   }
+  handleProjectDefState = (e) => {
+    const selected = e.target.attributes.value.value;
+    this.setState({ project_definition: selected });
+  };
+
+  renderProjectDefs = () => {
+    getConfigProjectDefinitions(this.props.location.pathname).then((res) =>
+      this.setState({
+        ProjectDefs: res.map((response) => (
+          <Dropdown.Item
+            key={response.name}
+            value={response.name}
+            active={response.name === this.state.state}
```

Comment:
I think you need an extra state property to store the selected project, `this.state.state` seems odd.